manifests/00-custom-resource-definition.yaml

```diff
@@ -1255,6 +1255,76 @@ spec:
                         type: string
                     type: object
                 type: object
+              idleConnectionTerminationPolicy:
+                default: Deferred
```

Comment:
This is a critical difference from #1182.  However, my understanding is that this difference is intentional.  @alebedev87, please confirm that the change is intentional **and** has a release note, and then cancel my hold.  

/hold

Reply:
I can answer my own question:

* The 4.18.z release note for https://issues.redhat.com//browse/OCPBUGS-48377 has, "Upgrading to this version preserves the existing behavior due to the default value, Deferred."
* The 4.19.0 release note for https://issues.redhat.com/browse/OCPBUGS-43745 has, "Upgrading to this version changes the existing behavior to Immediate."

I want us to follow up with docs to call that change in default behavior out more prominently if possible.  However, we do have a release note, so I'll cancel the hold.  

/hold cancel